### PR TITLE
Fix Badly Broken Links Pt 2

### PR DESCRIPTION
**Pull request by JRAlexander**
Second batch of updated links from conversion

## Summary

Second batch of updated links from conversion (part 2 of badly converted links)

Note: this PR is Part 2 of several to make fixing  #2459 easier to review. Part 1 was #3757.

Migration broken links are identified by searching the repo for *qualifyHint*, which can be found [here](https://github.com/dotnet/docs/search?utf8=%E2%9C%93&q=qualifyhint&type=).

[Internal Review URL (PrintByteArrayProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintByteArrayProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintDateTimeProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintDateTimeProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintInt32Property)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintInt32Property?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintJobPriorityProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintJobPriorityProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintJobStatusProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintJobStatusProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintPortProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintPortProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintProcessorProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintProcessorProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintQueueAttributeProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintQueueAttributeProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintQueueProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintQueueProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintQueueStatusProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintQueueStatusProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintServerLoggingProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintServerLoggingProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintStreamProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintStreamProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintStringProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintStringProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintSystemTypeProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintSystemTypeProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (PrintThreadPriorityProperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.PrintThreadPriorityProperty?view=netframework-4.7&branch=pr-en-us-3815)
[Internal Review URL (HtmlFormParameterWriter)](https://review.docs.microsoft.com/en-us/dotnet/api/system.web.services.protocols.htmlformparameterwriter?view=netframework-4.7&branch=pr-en-us-3815)

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: includes/note-compatibility-md.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
->  Functions and objects in the N:Microsoft.VisualBasic.Compatibility.VB6?qualifyHint=False&autoUpgrade=True namespace are provided for use by the tools for upgrading from Visual Basic 6.0 to Visual Basic 2008. In most cases, these functions and objects duplicate functionality that you can find in other namespaces in the .NET Framework. They are necessary only when the Visual Basic 6.0 code model differs significantly from the .NET Framework implementation.
\ No newline at end of file
+>  Functions and objects in the N:Microsoft.VisualBasic.Compatibility.VB6 namespace are provided for use by the tools for upgrading from Visual Basic 6.0 to Visual Basic 2008. In most cases, these functions and objects duplicate functionality that you can find in other namespaces in the .NET Framework. They are necessary only when the Visual Basic 6.0 code model differs significantly from the .NET Framework implementation.
```
Comment (mairaw) [inline includes/note-compatibility-md.md]: missing the xref here

File: xml/System.Web.Services.Protocols/HtmlFormParameterWriter.xml [hunk 2]
```diff
@@ -122,7 +122,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Web.Services.Protocols.HtmlFormParameterWriter.WriteRequest%2A> method calls the <xref:System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode%2A> writer,object[] values)?qualifyHint=True&autoUpgrade=True method.  
+ The <xref:System.Web.Services.Protocols.HtmlFormParameterWriter.WriteRequest%2A> method calls the <xref:System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode(System.IO.TextWriter,System.Object[])?displayProperty=nameWithType> method.
```
Comment (mairaw) [inline xml/System.Web.Services.Protocols/HtmlFormParameterWriter.xml]: given that the original was an autoUpgrade=true (which in the old days meant you wanted to link to the overload page),  you want to link to the * version

so change it to System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode%2A?displayProperty=nameWithType

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM. A few comments to be addressed.

**Comment by mairaw** [comment]
Do you need to rename the PR to remove the WIP check @JRAlexander?

**Comment by JRAlexander** [comment]
Renamed it and removed wip label

**Review by mairaw — APPROVED** [review]
LGTM

**Pull request merged**